sdk/samples/iot/paho_iot_hub_methods_sample.c

```diff
@@ -211,41 +213,43 @@ static void receive_method_messages(void)
   // Continue until max # messages received or timeout expires.
   for (uint8_t message_count = 0; message_count < MAX_METHOD_MESSAGE_COUNT; message_count++)
   {
-    LOG(" "); // Formatting
-    LOG("Waiting for Method message.\n");
+    IOT_SAMPLE_LOG(" "); // Formatting
+    IOT_SAMPLE_LOG("Waiting for Method message.\n");
 
     if (((rc
           = MQTTClient_receive(mqtt_client, &topic, &topic_len, &message, MQTT_TIMEOUT_RECEIVE_MS))
          != MQTTCLIENT_SUCCESS)
         && (rc != MQTTCLIENT_TOPICNAME_TRUNCATED))
     {
-      LOG_ERROR("Failed to receive message #%d: MQTTClient return code %d.", message_count + 1, rc);
+      IOT_SAMPLE_LOG_ERROR(
+          "Failed to receive message #%d: MQTTClient return code %d.", message_count + 1, rc);
       exit(rc);
     }
     else if (message == NULL)
     {
-      LOG_ERROR("Timeout expired: MQTTClient return code %d.", rc);
+      IOT_SAMPLE_LOG("Receive message timeout expired.");
       return;
     }
     else if (rc == MQTTCLIENT_TOPICNAME_TRUNCATED)
     {
       topic_len = (int)strlen(topic);
     }
-    LOG_SUCCESS("Message #%d: Client received a message from the service.", message_count + 1);
+    IOT_SAMPLE_LOG_SUCCESS(
+        "Message #%d: Client received a Method message from the service.", message_count + 1);
 
     // Parse method message and invoke method.
     az_iot_hub_client_method_request method_request;
     parse_method_message(topic, topic_len, message, &method_request);
```

Comment:
sorry not part of this pr but
```suggestion
    parse_method_request(topic, topic_len, message, &method_request);
```

Reply:
it still fits.  all part of finalizing the samples.

Reply:
hmmm, after looking at it, I think it should stay with message.  The message is what is being parse, not the method_request.  The method_request will be initialized from info parsed from the message.